src/include/miopen/solver/implicitgemm_ck_util.hpp

```diff
@@ -558,9 +558,9 @@ inline size_t GetCKAlphaBetaWorkspace(const miopen::conv::ProblemDescription& pr
 inline bool CKWrwRequireWorkspace(
     size_t G, size_t C, size_t K, miopenDataType_t data_type, miopenAlphaBetaCase_t alpha_beta_case)
 {
-    auto is_odd     = [](int num) { return num % 2 != 0; };
-    int C_per_group = C / G;
-    int K_per_group = K / G;
+    auto is_odd        = [](size_t num) { return num % 2 != 0; };
+    size_t C_per_group = C / G;
+    size_t K_per_group = K / G;
```

Comment:
We do not need to check `==1` because guess what `1` is odd :).

Reply:
For example if `C` = 64 and `G` = 2, `C_per_group` = 32. So, C_per_group can be any number.

Reply:
It is enough to check if C_per_group or K_per_group are odd. From CK point of view we doesnt care about C and K. It is only about C_per_group or K_per_group .

Reply:
OK, yeah, I didn't notice that. Good catch @bartekxk . @bghimireamd you need to input `C_per_group` to `isOdd`, not `C`.